test-common/generators/ModelGenerators.scala

```diff
@@ -288,6 +289,87 @@ trait ModelGenerators {
     } yield models.sdes.File(recipientOrSender, name, location, checksum, size, properties)
   }
 
+  implicit val arbitraryReportType: Arbitrary[ReportType] = Arbitrary {
+    Gen.oneOf(ReportType.values)
+  }
+
+  implicit val arbitraryFileStatus: Arbitrary[FileStatus] = Arbitrary {
+    for {
+      status <- Gen.oneOf(models.submission.Pending, models.submission.Accepted, models.submission.RejectedSDES, models.submission.RejectedSDESVirus)
```

Comment:
Perhaps a `FileStatus.values` in the `FileStatus` object to have it all in one location for management/maintenance reasons.